templates/.walg.json.j2-backup

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
".walg.json.j2-backup" file here is superfluous

Reply:
i will remove .walg.json.j2-backup Later